just/bluefin-tools.just

```diff
@@ -56,6 +56,8 @@ ollama:
             ;;
     esac
 
+    mkdir -p "$HOME/.ollama"
```

Comment:
Perhaps this would be better suited in the systemd unit itself? 

Untested, but likely look something like this (note the second ExecStartPre):

```
    [Service]
    Restart=always
    TimeoutStartSec=60
    # Ensure there's a userland podman.sock
    ExecStartPre=/bin/systemctl --user enable podman.socket
    ExecStartPre=-mkdir -p %h/.ollama
```

Reply:
Yes, putting into the service is better.
Don't you mean:

```bash
ExecStartPre=mkdir -p %h/.ollama
```

I don't understand the `-` in `-mkdir`, was that a typo?